pandas/core/internals.py

```diff
@@ -1696,7 +1696,7 @@ def _try_coerce_args(self, values, other):
             other = other.value
         elif isinstance(other, np.timedelta64):
             other_mask = isnull(other)
-            other = other.view('i8')
+            other = Timedelta(np.timedelta64(other))
```

Comment:
Can you address this one? 
The `np.timedelta64(..)` is not needed given you just tested that it is already a `np.timedelta64` object